public/locales/fr/common.json

```
@@ -0,0 +1,56 @@
+{
+  "scrobble": "Scrobble",
+  "getScrobbling": "Début du scrobbling",
+  "loading": "Chargement...",
+  "artist": "Artiste",
+  "song": "Musique",
+  "title": "Titre",
+  "album": "Album",
+  "albumArtist": "Artiste de l'album",
+  "history": "Historique",
+  "yourProfile": "Votre profil",
+  "language": "Langage",
+  "translateYourLanguage": "Vous voulez ajouter votre langage?",
+  "logIn": "Se connecter",
+  "logOut": "Se déconnecter",
+  "now": "Maintenant",
+  "custom": "Personnaliser...",
+  "timestamp": "Timestamp",
+  "lastfmWillRejectOldTimestamps": "Last.fm rejecte les scrobbles datant de plus de deux semaines.",
+  "noSongsScrobbled": "Aucune musique n'a été scrobblé !",
+  "songsWillAppearHere": "Les pistes apparaîtront ici une fois que vous les aurez scrobblés.",
+  "clearHistory": "Nettoyer l'historique ",
```

Little fix: remove trailing space

```suggestion
  "clearHistory": "Nettoyer l'historique",
```